### ISSUE TITLE
notebook gets renamed after last note is moved

### ISSUE THREAD

**Issue by Geiren** [neilagabriel/vim-geeknote#22]
Steps to reproduce:
  * Create 2 notebooks, for example "unsorted" and "archived"
  * Create notes in unsorted
  * In the navigation window it will say "unsorted (2)"
  * After moving the first note to another notebook, it will say "unsorted (1)", and it does not rename it
  * After moving the last note in "unsorted" it will rename then notebook to "unsorted (1)" literally.

This is nothing major so don't stress it :)

**Comment by neilagabriel** [comment]
Yuck.

Yes, I can see how that could happen. I tried to code this in a way that
would allow the user to have " (#)" appended to the name of a notebook. I
don't why you would do that, but I didn't want to disallow it. When looking
to see if the user renamed a notebook in the navigation window, I need to
discern between the " (#)" that the user added vs. the " (#)" that the
plugin adds to non-empty notebooks. I use the notebook's note count.
Obviously if it is empty then the " (#)" must have been added by the user.
Or at least I thought :) ...

Thanks for pointing it out. I'll work on a fix.
